src/serac/physics/tests/solid.cpp

```diff
@@ -320,7 +320,7 @@ void functional_parameterized_solid_test(double expected_disp_norm)
   EXPECT_NEAR(expected_disp_norm, norm(solid_solver.displacement()), 1.0e-6);
 }
 
-TEST(SolidMechanics, 2DQuadParameterizedStatic) { functional_parameterized_solid_test<2, 2>(2.1906312704664623); }
+TEST(SolidMechanics, 2DQuadParameterizedStatic) { functional_parameterized_solid_test<2, 2>(2.1773851975471392); }
```

Comment:
any idea what's causing the ~1% change in solution norm?

Reply:
I believe it's because the AMG preconditioner isn't as accurate anymore and it's hard to force the linear solve to continue iterating past this point.